ansible/playbooks/roles/grafana/tasks/upgrade/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Include new version of defaults
+  include_vars:
+    file: defaults/versions.yml
+    name: grafana_versions
+
+- name: Include upgrade defaults
+  include_vars:
+    file: defaults/upgrade.yml
+    name: upgrade_defaults
+
```

Comment:
For consistency, I would use the same approach as we have for `opendistro_for_elasticsearch` or `kafka_exporter` adding such task:

```
- name: Assert that grafana package is installed
  assert:
    that: ansible_facts.packages['grafana'] is defined
    fail_msg:
      - grafana package not found, nothing to upgrade.
      - Re-apply your configuration or remove grafana group from Ansible inventory.
    quiet: true
```

Then some logic below could be simplify.